HomeworkJava8/Java8Assignament.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * in this application we have to use stream and lambda expression to read a csv file,
+ * target a month in the file from user,
+ * and write back another csv file with the name alphabetically for the matches which have the birthday on a month indicated
+ * @author Iagar Denisa
+ */
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class Java8Assignament {
+    String inputFile;
+    String outputFile;
+    int month;
+
+    public Java8Assignament(String inputFile, String outputFile, int month) {
+        this.inputFile = inputFile;
+        this.outputFile = outputFile;
+        this.month = month;
+    }
+
+    public List<String> readFile() {
+        this.inputFile = "HomeworkJava8\\InputText\\Namelist.csv";
```

Comment:
since you get this value as a constructor parameter, why are you setting the value here?